lib/livebook/settings.ex

```diff
@@ -18,6 +18,22 @@ defmodule Livebook.Settings do
     end
   end
 
+  @doc """
+  Sets the current autosave path.
+  """
+  @spec set_autosave_path(String.t()) :: :ok
+  def set_autosave_path(autosave_path) do
+    storage().insert(:settings, "global", autosave_path: autosave_path)
```

Comment:
Once we have any other settings under global we would override. Maybe we should have an option for merge-like behaviour (where we don't remove existing entries), or a separate `insert_key` function. Theoretically we could do `insert(:settings, "autosave_path", value: autosave_path)`, but this doesn't feel right. @josevalim thoughts?

Reply:
The `insert` already works in merge like behaviour, the attributes get accumulated throughout the following inserts and can be deleted all at once with `delete` so the insert here will just override the old `auto_path` value 😉

Reply:
Ahh, we're deleting only the keys that we are about to insert, perfect!